src/behavior-considered-undefined.md

```diff
@@ -34,6 +34,7 @@ code.
 * Invoking undefined behavior via compiler intrinsics.
 * Executing code compiled with platform features that the current platform
   does not support (see [`target_feature`]), *except* if the platform explicitly documents this to be safe.
+* Performing non-nop coercion on a dangling or unaligned raw pointer.
```

Comment:
Though now that I read this, I think this should be clarified though to integrate language which more precisely reflects the decision in rust-lang/rust#101336. 

Specifically, should we mention that the *vtable pointer* needs to point to a valid vtable?